### PR TITLE
Added recipe for PySocks

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipes/pysocks/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - socks
+
+about:
+  home: https://github.com/Anorov/PySocks
+  license: BSD 3-Clause
```
Comment (jakirkham) [inline recipes/pysocks/meta.yaml]: Please add license file.

Comment (pmlandwehr) [inline recipes/pysocks/meta.yaml]: There's no MANIFEST file or mention of a license. Submitted an [issue](https://github.com/Anorov/PySocks/issues/49)

Comment (jakirkham) [inline recipes/pysocks/meta.yaml]: Ok, please just add a comment in the recipe with a reference to that issue for now.

**Comment by msarahan** [comment]
LGTM.  Please ping the author in an issue on the feedstock and invite them to be maintainers if they'd like.

**Pull request merged**